### PR TITLE
build(client,build-common,tinylicious): Enable prettier and format code

### PR DESCRIPTION
**Pull request by jikim-msft**
#### Description

This PR changes the `lint` & `lint:fix` scripts to add prettier to the lint tasks. It also adds a `format` script to all projects. Finally, `lint:fix` was executed on all projects.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6640,6 +6640,12 @@
               }
             }
           }
+        },
```
Comment (Josmithr) [inline package-lock.json:6643]: We didn't make any dep changes to the root (client release group's) package.json, so I don't think these changes should be here.

**Review by Josmithr — COMMENTED** [review]
Looks good, aside from the package-lock changes!

So now I think we just need to coordinate with the rest of the group in terms of timing, merge any new changes from main into your branch and run the formatter!

File: package-lock.json (hunk 1 above)
Comment (jikim-msft) [reply package-lock.json]: Reverted `package-lock.json`

**Review by Josmithr — APPROVED** [review]
Since the majority of this is just formatting changes, I'm approving. I guess we just need to keep an eye on pipelines and notify the team once this goes in?

**Pull request merged**